paimon-web-server/src/main/java/org/apache/paimon/web/server/service/impl/MetadataServiceImpl.java

```diff
@@ -171,6 +172,26 @@ public List<DataFileVO> getDataFile(MetadataDTO dto) {
         return result;
     }
 
+    @Override
+    public List<OptionVO> getOption(MetadataDTO dto) {
+        initEnvironment(dto, MetadataConstant.OPTIONS);
+
+        List<OptionVO> result = new LinkedList<>();
+
+        try {
+            reader.forEachRemaining(
+                    internalRow -> {
+                        OptionVO optionVo = new OptionVO();
+                        optionVo.setKey(internalRow.getString(0).toString());
```

Comment:
Is there any `NullPointerException` when `internalRow.getString(0)` is null?

Reply:
Thanks for the review @SteNicholas . Maybe I did not think it through thoroughly, I cannot anticipate under what circumstances the $options table would have a null value.